x-pack/platform/test/security_functional/login_selector.config.ts

```diff
@@ -85,6 +85,7 @@ export default async function ({ readConfigFile }: FtrConfigProviderContext) {
               realm: 'saml1',
               description: 'Log-in-with-SAML',
               icon: 'logoKibana',
+              origin: getUrl.baseUrl(kibanaFunctionalConfig.get('servers.kibana')),
```

Comment:
__nit:__ what do you think about adding `origin` in an array format to another provider too, so that we have all four cases covered (no origin, matching origin as a string, matching origin as an element of an array of different origins, and origin that doesn't match)?